tests/unit_tests/Test/PM/Unit/SemVer/Expressions.cls

```diff
@@ -34,13 +34,25 @@ Method TestPrerelease()
 	Do ..AssertSatisfied(">=1.0.0-beta.1","1.0.0-beta.1")
 	Do ..AssertSatisfied(">=1.0.0-beta.1","1.0.0-beta.2")
 	Do ..AssertSatisfied(">=1.0.0-alpha.1","1.0.0-beta.1")
-	Do ..AssertSatisfied("^1.0.0","1.0.0-1.m1")
+	Do ..AssertNotSatisfied("^1.0.0","1.0.0-1.m1") // This shouldn't be satisfied because 1.0.0-1.m1 is a prerelease
```

Comment:
@isc-tleavitt @isc-shuliu  This may cause us some problems with maintenance releases where we currently rely on this behavior. If we don't want to support this, our release tool will need to lock down versions in not just the base module but every dependency.
For example, HealthConnect 7.0.0 lists HS.REST 1.1.0-1.m1 and HealthConnect.Common 4.1.1-1.m1 as fixed version dependencies. However, HealthConnect.Common lists HS.REST ^1.0.0 as a dependency. This would now break unless HealthConnect.Common also lists a specific version as a dependency.
We need the ability to toggle this behavior on vs off I think for HS. Either that or come up with an alternate solution for maintenance releases.
cc @isc-eneil

Reply:
@isc-kiyer that's poor use of semantic versioning to say that a prerelease is sequentially after a main release. If there's an alternate solution for maintenance releases that would be preferable.

Reply:
Specifically, I think this comes down to discipline around saying what's really a bugfix vs. enhancement, and making decisions about releases of individual packages based on their contents. Under what circumstances would we not want a top-level release to include fixes to known bugs in a dependency under a given minor release?

Reply:
All that said, we could address this with a top-level command and configurable "include prerelease" switch similar to includePrerelease in node-semver.

Reply:
So question is, among the following options, where does an "include prerelease" switch belong?
* module.xml, as a subelement of the module (or could *perhaps* jump through hoops in XSLTs to support via an attribute of Dependencies) - use case: always make it explicit for top-level packages in our package-lock-ish use case for e.g. HealthConnect, HSBASE.
* option in "load" and "install" commands - use case: in a proper SemVer environment or possibly a CI workflow, pick up not-yet-released upcoming versions of dependencies for testing
* system-level setting - use case: development environments where we'll always want to be able to pick up prereleases

My gut feeling is the system level setting, because it'll cover all cases and I want to do as little work as possible to enable poor use of SemVer.

Reply:
I think a system level setting is a good idea. That being said, I would like to perhaps revisit just taking the next patch version (even if including other patches along the way) with the HS team to see if we can change maintenance releases to do that instead. Copying @bourette

Reply:
I raised this with Jaideep, perhaps we could do the same with I4H/HC.

Reply:
Just for completeness, we moved from + to - when we realized that re-spinning a release caused it to pick up newer code, making it impossible to reproduce a kit.